src/stories/Blocks/content-list-page/content-list-page.scss

```diff
@@ -19,6 +19,7 @@
   }
 }
 
+//  Todo should be merged with .facet-line
```

Comment:
Just a comment: I don't buy into the "no todos in code!" mindset, but: a todo should either explain what and why something needs to be done or link to an issue that explains it. Otherwise, it becomes a personal todo that only you know the reason for. And my personal experience is that, two weeks from now, I can't remember why I thought they should be merged.

If the same todo applies in multiple places, they can point to each other to avoid redundancy.